go/add_slashes.go

```diff
@@ -0,0 +1,16 @@
+package pehape
+
+func Addslashes(str string) string {
```

Comment:
Your function name should be `AddSlashes`, I think. Also, please add docstring.

For reference: https://go.dev/blog/godoc